int16 smoke_val